### ISSUE TITLE
Add Pokémon sprites

### ISSUE THREAD

**Issue by digitalghost-dev** [digitalghost-dev/poke-cli#93]
`v0.11.0` will include a new flag to the `pokemon` command that will generate an image of a selected Pokémon.

**Comment by digitalghost-dev** [comment]
@scottmckendry - I've add my first draft of using an `--image | -i` flag with the `pokemon` command.

Related files are `cmd/pokemon.go` and `flags/pokemonflagset.go`

example: `poke-cli pokemon bulbasaur --image`

I am able to get an image rendered but would like the option to select a size. For example, `--image=lg` and this can update the dimensions when calling the `ToString()` func.

```go
imgStr := ToString(60, 60, img)
```

**Comment by ancientcatz** [comment]
> `v0.11.0` will include a new flag to the `pokemon` command that will generate an image of a selected Pokémon.

I suggest considering [`github.com/spf13/pflag`](https://github.com/spf13/pflag). It supports one-letter shorthand flags, which can enhance usability. It also works as a drop-in replacement for the standard `flag` package, making it easier to adopt.

**Comment by digitalghost-dev** [comment]
Yeah, I've been planning on doing this and aiming to implement this for `v0.12.0`! I just wanted to get the images working which I think I have now.
